en, zh: add docs of topology spread constraints and security context (#1184)
pingcap/docs-tidb-operator: en/configure-a-tidb-cluster.md

### High availability of TiDB service

#### Use affinity to schedule pods

> By configuring `PodAntiAffinity`, you can avoid the situation where different instances of the same component are deployed on the same physical topology node. In this way, disaster recovery is achieved. For the user guide of Affinity, see [Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity).

Reviewer: ```suggestion
By configuring `PodAntiAffinity`, you can avoid the situation in which different instances of the same component are deployed on the same physical topology node. In this way, disaster recovery (high availability) is achieved. For the user guide of Affinity, see [Affinity & AntiAffinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity).
```

       - ${namespace}
```

#### Use topologySpreadConstraints to make pods evenly spread

> By configuring `topologySpreadConstraints`, you can make pods evenly spread in different topologies. For instructions about configure `topologySpreadConstraints`, see [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/).

Reviewer: ```suggestion
By configuring `topologySpreadConstraints`, you can make pods evenly spread in different topologies. For instructions about configuring `topologySpreadConstraints`, see [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
```

> **Note:**
>
> To use `topologySpreadConstraints`, you must enable the `EvenPodsSpread` feature gate. If the Kubernetes version in use is earlier than v1.16 or if the `EvenPodsSpread` feature gate is disabled, the configuration of `topologySpreadConstraints` does not take effect.

> You can either configure `topologySpreadConstraints` at a cluster level (`spec.topologySpreadConstraints`) for all of components or at a component level (such as `spec.tidb.topologySpreadConstraints`) for a specific component.

Reviewer: ```suggestion
You can either configure `topologySpreadConstraints` at a cluster level (`spec.topologySpreadConstraints`) for all components or at a component level (such as `spec.tidb.topologySpreadConstraints`) for specific components.
```
